ESP32LapTimer/src/HardwareConfig.h

```diff
@@ -31,7 +33,7 @@
 // BELOW ARE THE ADVANCED SETTINGS! ONLY CHANGE THEM IF YOU KNOW WHAT YOUR ARE DOING!
 
 #define EEPROM_VERSION_NUMBER 9 // Increment when eeprom struct modified
-#define MaxNumReceivers 6
+#define MaxNumReceivers 8
```

Comment:
The channels are also hardcoded on the webpage, so that needs to be dynamic as well

Reply:
This i had not considered and ill have to see if that can be solved in a resonably simple way.